draft-lmbdhk-quic-multipath.md

```diff
@@ -610,7 +610,14 @@ be controlled by the combination of one or several of the following:
    uses a series of consecutive sequence numbers without creating
    holes.
 
-
+### ACK Delay Considerations
+
+The ACK Delay field of the ACK frame is relative to the largest
+acknowledged packet number (see {{Section 13.2.5 of QUIC-TRANSPORT}}).
+When using paths with different transmission delays, the reported host
+delay will mostly relate to the path with the shortest latency. To
```

Comment:
Maybe I'm missing something but why will it _mostly_ relate to the shortest path? Isn't it kind of random which path the highest ack'ed packet belongs to?

Reply:
Consider a scenario with two paths, A and B, where the latency of A is larger than the B's one. The sender wants to send two packets, spread on both paths. We have two cases.

1. 1 sent on B, 2 sent on A

There are two sub-cases here.

- The receiver sends an ACK before having received 2. `ACK(largest=1)` contains the ACK delay relative to path B. Then it receives 2, and sends `ACK(largest=2)` with ACK delay relative to path A. In this case, we have ACK delay for both paths.

- The receiver waited for 2 before sending its ACK. Then, the ACK delay of `ACK(largest=2)` relates to path A, and we don't have the host delay of path B.

2. 1 sent on A, 2 sent on B

While we still have the two sub-cases, they both lead to the same result: `ACK(largest=2, missing=1)` and `ACK(largest=2)` have their ACK delay relative to the host delay of path B and not to the path A.

In general, if the largest packet number has been sent on the lowest latency path, it will prevent the receiver from advertising the host delay of higher latency paths. Note that the "ideal case" (where both paths have estimates) only works if the sender does not transmit further packets on the fast path, in which case we fall into case 2. As the fast path will get its packets acknowledged quicker, the sender will likely send further packets on fast path first. It might still be possible to get host delay estimates of slower paths, but most of the ACK delays will relate to the fast one.

Reply:
Ah okay, the assumption is that you would basically receive all packets on the slow path "out-of-order". Got it now. Thanks!

Reply:
I think the word "mostly" is a bit confusing. It could be that when someone uses a round-robin scheduler, unfortunately, it always measures the delay of the high latency path.

Reply:
Would replacing "mostly" by "most of the time" be better?